bundles/automation/org.eclipse.smarthome.automation.module.script.defaultscope/src/main/java/org/eclipse/smarthome/automation/module/script/defaultscope/internal/ItemRegistryDelegate.java

```diff
@@ -62,7 +62,11 @@ public boolean containsValue(Object value) {
 
     @Override
     public State get(Object key) {
-        return itemRegistry.get((String) key).getState();
+        final Item item = itemRegistry.get((String) key);
+        if (item == null) {
+            return null;
```

Comment:
What should we do if the item is not known (null or undef state, null, illegal argument exception)?

Reply:
The Javadoc says `@return element or null if no element was found`, so null should be just right.